### PR TITLE
Internationalization support

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

**Comment by bacongobbler** [comment]
Hey @lucperkins, where are we on this PR? Is this ready for review or still a WIP?

**Comment by lucperkins** [comment]
@bacongobbler Still plugging away

**Comment by lucperkins** [comment]
@bacongobbler Okay, I take it back. There's still more to be done here, but this puts us in a position where people can start to add documentation translations. Feel free to review.

File: themes/helm/layouts/partials/nav.html [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
+{{ range $menu }}
+<li>
+  <a href="{{ .URL | relLangURL }}">
+    {{ .Name  }}
+  </a>
+</li>
+{{ end }}
+
+
+{{ range .Translations }}
```
Comment (flynnduism) [inline themes/helm/layouts/partials/nav.html]: Can we hide the `/zh` link until we have Chinese content?

**Comment by lucperkins** [comment]
@flynnduism FYI this is ready for review

File: themes/helm/layouts/partials/banner.html [hunk 2]
```diff
@@ -8,9 +8,24 @@
     <!-- <p class="center show-for-small-only"><a href="https://helm.sh/docs"><small style="font-size: 0.9rem;">You are viewing Helm 2 (latest). &nbsp; Helm 3 is here - <a href="https://v3.helm.sh" target="_blank"><strong>Docs</strong></a> | <a href="https://helm.sh/blog/" target="_blank"><strong>Blog</strong></a></small></a></p> -->
 
     <!-- viewing helm 3 version of site (desktop) -->
+<<<<<<< HEAD
```
Comment (flynnduism) [inline themes/helm/layouts/partials/banner.html]: Some rebasing issues here in `banner.html`

**Comment by bacongobbler** [comment]
can you please rebase this against master? We just switched branches over in helm/helm so I'd like to make sure this branch has all those changes present. Thanks!

**Comment by lucperkins** [comment]
@flynnduism Fixed.

@bacongobbler I just tried locally and I'm actually not sure I'm competent to merge with `master`. Could we possibly merge this into `dev-v3-docs` and allow someone more intimately acquainted with the codebase to attempt to merge?

**Comment by bacongobbler** [comment]
Sorry, minor typo on my end. I did mean to ask you to rebase against the dev-v3 docs branch, not master.

**Comment by lucperkins** [comment]
@bacongobbler Okay, I rebased against `dev-v3-docs`. It may *seem* not-yet-rebased but the changeset is legitimately massive, as internationalization involves changing the whole directory structure of `content`.

**Comment by bacongobbler** [comment]
it also looks like we have duplicate content in both content/docs and content/en/docs. Perhaps some things were not moved over in the last rebase?

**Comment by lucperkins** [comment]
@bacongobbler Okay, I removed those potentially redundant directories.

@flynnduism Could you have a look and make sure that's the right call?

**Comment by bacongobbler** [comment]
those documents still exist under content/en/docs.

Additionally, some content was accidentally removed, namely the Role-based access control and the v2 to v3 migration docs. Would you mind adding those back?

**Comment by bacongobbler** [comment]
Hey @lucperkins, I took what you have over here, cleaned it up a bit and PR'd #318 to address some of the concerns I had with the current PR, while still moving forward with the same approach. I'd love to hear your feedback on that PR, in particular with the homepage change that moves the main content under the internationalization pages.

**Comment by flynnduism** [comment]
Luc's work on the _**theme**_ side is ✅.

IMO it'll be more straight-forward if @bacongobbler and I to finalize and rebase the _content_ side o things, so much has been in flux on this project. I'm free to tackle this today.

**Comment by flynnduism** [comment]
I tried rebasing and pushing to luc's branch here, but got stuck. Hence I've opened this in another PR with the final small updates needed to this to get this into `dev-v3-docs` for launch.

* rebased
* added recent blog and doc edits `content/en/`
* updated readme

Closing this off, will ship the i18n work via #374.

**Comment by lucperkins** [comment]
@flynnduism That’s the right call. I would’ve done the same thing 👍🏼

**Pull request closed without merging**